app/src/test/java/mozilla/lockbox/store/ClipboardStoreTest.kt

```diff
@@ -48,15 +48,15 @@ class ClipboardStoreTest : DisposingTest() {
     fun testCopyUsername() {
         val testString = "my_test_string"
         dispatcher.dispatch(ClipboardAction.CopyUsername(testString))
-        val clip = clipboardManager.primaryClip.getItemAt(0)
+        val clip = clipboardManager.primaryClip?.getItemAt(0) ?: throw AssertionError("PrimaryClip must not be null")
```

Comment:
I don't think these `throw`s are necessary here since we know that the clipboard contains an item.

Reply:
Sure we know that it wont be null. But Kotlin does not know this. So either we @suppress the warning or we do this.

I just feel uncomfortable supressing the warning if we have an alternative. But tell me what you think about that!

Reply:
@beatbrot I approve of your comment.

Reply:
@jhugman Sorry if this question sounds stupid, but do you work at Mozilla and does your approval mean, that I can view this finding as resolved?

Reply:
Yes, we both approve :) Thanks for the explanation!